### ISSUE TITLE
Autoconfiguration ipv4 on wg server networkcard its getting random ip addres not gatway of the range of ip i set up

### ISSUE THREAD

**Issue by gbtech-org** [micahmo/WgServerforWindows#118]

**Comment by micahmo** [comment]
Hi @gbtech-org,

Just to make sure I understand the problem, are you using NAT Routing? And are you saying that the IP address assigned to the `wg_server` interface is not the first address in the server address range?

![image](https://github.com/micahmo/WgServerforWindows/assets/7417301/7f5d124b-b397-4de8-8583-6ab1da887096)

![image](https://github.com/micahmo/WgServerforWindows/assets/7417301/2084d52f-3b63-45cf-be94-4a19fbd8fd6e)

**Comment by gbtech-org** [comment]
Yes i have set up correctly but i dont know why the wg-server get an random ip apipa

**Comment by micahmo** [comment]
You're welcome. Those should be the exact commands that WS4W runs. You can see the code.

https://github.com/micahmo/WgServerforWindows/blob/913f6a298f5d285a5bc62ec728a18116c987c648/WgServerforWindows/Models/NewNetNatPrerequisite.cs#L88

https://github.com/micahmo/WgServerforWindows/blob/913f6a298f5d285a5bc62ec728a18116c987c648/WgServerforWindows/Models/NewNetNatPrerequisite.cs#L94

**Comment by glockmane** [comment]
I have a similar Problem. NAT is disabled and I get a 169... IP Adress on the WireGuard Adapter.. Then I manually change it to 10.253.0.1 but after Restart (guess the Interface is removed and created again each reboot) I have the same problem.. I don't want Clients to have Network and Internet Access, they should only have access to the WireGuard Server Computer..

**Comment by micahmo** [comment]
Hey, just wanted to follow up here.

@gbtech-org, did we solve your issue by running the PowerShell command?

@glockmane, are you still having an issue? I think it might be slightly different than the OP issue. I think you just want WS4W to optionally stop assigning an IP to your interface right?

**Comment by gbtech-org** [comment]
Hello @micahmo yes the powershell fixed my issue
